.github/workflows/build-and-test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     name: Erlang ${{matrix.otp}}
     strategy:
       matrix:
-        otp: ['22.3.4.20', '23.3.4.7', '24.1']
+        otp: ['23.3.4.7', '24.1']
```

Comment:
Don't we support 22 anymore?

Reply:
There's a problem with building property based tests on 22 - https://github.com/erszcz/Gradualizer/runs/3983416094?check_suite_focus=true. I rechecked with the latest version of this branch again, just to make sure, but no luck - https://github.com/erszcz/Gradualizer/runs/4056122650?check_suite_focus=true. The only info about this I could find is http://erlang.org/pipermail/erlang-questions/2011-June/059722.html, but it's a dead end.

Reply:
I don't know if we need CI on OTP 22. Maybe it's better to have it if it's easy? Is it hard to just exclude the property tests on the OTP 22 build?

Reply:
Ok, it seems we can quite easily skip property tests on 22, but have that version in the matrix - done in the latest commit.